flak-backend-jdk/src/main/java/flak/backend/jdk/JdkApp.java

```diff
@@ -133,6 +135,11 @@ void on404(JdkRequest r) throws IOException {
     }
   }
 
+  @Override // unlock package-private access
```

Comment:
Looks like this is not needed since the method is already protected in AbstractApp

Reply:
It's because this method is called from `Context`, so it's using the package-private access level, which require the method to be re-defined (so that it's in the same package): `Context.java:67: error: onBefore(SPRequest) has protected access in AbstractApp`

https://github.com/pcdv/flak/blob/c80ac185e259418cae65de798b36bd5c34240de1/flak-backend-jdk/src/main/java/flak/backend/jdk/Context.java#L67